### ISSUE TITLE
Foreign language audio stream on certain YouTube videos 

### ISSUE THREAD

**Issue by Taratect** [TeamNewPipe/NewPipe#9804]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.25.0

### Steps to reproduce the bug

Watch the url below on newpipe
https://www.youtube.com/watch?v=TJ2ifmkGGus
https://m.youtube.com/watch?v=frYegVZrWzg

Basically the last 5 videos of MrBeast YouTube channel

### Expected behavior

The audio stream should be in English by default 

### Actual behavior

The audio stream is in different foreign language 

### Affected Android/Custom ROM version

Android 10, MIUI 12

### Affected device model

Redmi 8

### Additional information

I think this is because google is tweaking with multiple audio streams in a single video
However newpipe should have systems in place so that the audio stream is in English by default.
And also have the settings to change the audio stream in the newpipe app.
